jupyterlab/handlers/yjs_echo_ws.py

```diff
@@ -0,0 +1,79 @@
+import uuid
```

Comment:
Maybe add docstring + copyright, as in other python modules, e.g. [extension_manager_handler.py](https://github.com/jupyterlab/jupyterlab/blob/f4691d7d114c706aba484effecfee703310e142d/jupyterlab/handlers/extension_manager_handler.py#L1-L4)?

Reply:
Thanks! 

Fixed in https://github.com/jupyter-rtc/jupyterlab/pull/74